### PR TITLE
Update models.py

### PR DESCRIPTION
**Pull request by nitish705702**
@Update models.py 
At least this should be as part of a function that can be called 
and tested so we can discover the changes.
: interior
: evolution
They should return the appropriate tables for each.
Moving forward they can become kind of attribute of the sun class like variable constant.

### CODE REVIEW

**Comment by nabobalis** [comment]
Sorry @nitish705702, we already have this open pull request addressing this issue (https://github.com/sunpy/sunpy/pull/2936).

I would suggest having a look at another issue to tackle!

**Comment by nitish705702** [comment]
@nabobalis Sir

**Comment by nabobalis** [comment]
@nitish705702 Sorry, I don't quite follow.

**Comment by nitish705702** [comment]
Any one can help me .how should  resolve this error  "Bash exited with code '1'.  "

**Comment by nabobalis** [comment]
> Please Any one can help me .how should resolve this error "Bash exited with code '1'. "

You are referring to the error on the Azure builds. If you go here (https://dev.azure.com/sunpy/sunpy/_build/results?buildId=736) and click on the "run tox" bar, it will open up a large log which has the error messages in it. 

From there you can find that the issue that caused the build to fail is:

```
====================================================================================== ERRORS ======================================================================================
_______________________________________________________________________ ERROR collecting sunpy/sun/models.py _______________________________________________________________________
.tox/py36-offline/lib/python3.6/site-packages/pytest_doctestplus/plugin.py:107: in collect
    module = self.fspath.pyimport()
.tox/py36-offline/lib/python3.6/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
E     File "/home/vsts/work/1/s/sunpy/sun/models.py", line 59
E       interior = pandas.DataFrame((pandas . _mass , pandas . _luminosity,pandas . _temperature, pandas . _density, pandas ._radius),(columns=['mass', 'luminosity', 'temperature','density' ,'radius'])
E                                                                                                                                             ^
E   SyntaxError: invalid syntax
_______________________________________________________________________ ERROR collecting sunpy/sun/models.py _______________________________________________________________________
.tox/py36-offline/lib/python3.6/site-packages/pytest_doctestplus/plugin.py:107: in collect
    module = self.fspath.pyimport()
.tox/py36-offline/lib/python3.6/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
E     File "/home/vsts/work/1/s/sunpy/sun/models.py", line 59
E       interior = pandas.DataFrame((pandas . _mass , pandas . _luminosity,pandas . _temperature, pandas . _density, pandas ._radius),(columns=['mass', 'luminosity', 'temperature','density' ,'radius'])
E                                                                                                                                             ^
E   SyntaxError: invalid syntax
```

So it seems that the two lines added are invalid Python syntax.

File: sunpy/sun/models.py [hunk 1]
```diff
@@ -56,13 +56,12 @@
            10.157, 5.566, 2.259, 0.4483, 0.1528,
            0.042, 0.00361, 1.99e-7]
 
-_d = {'mass': _mass, 'luminosity': _luminosity, 'temperature': _temperature, 'density': _density}
-interior = pandas.DataFrame(_d, index = _radius)
-interior.index.name = 'radius'
+interior = pandas.DataFrame((pandas . _mass , pandas . _luminosity,pandas . _temperature, pandas . _density, pandas ._radius),(columns=['mass', 'luminosity', 'temperature','density' ,'radius'])
```
Comment (nitish705702) [inline sunpy/sun/models.py]: interior = pandas.DataFrame((pandas . _mass , pandas . _luminosity,pandas . _temperature, pandas . _density, pandas ._radius),(columns=['mass', 'luminosity', 'temperature','density' ,'radius']))

Comment (nitish705702) [reply sunpy/sun/models.py]: i forgot ')'
this code.

File: sunpy/sun/models.py [hunk 2]
```diff
@@ -83,10 +82,8 @@
 _tcentral_temperature = [13.35, 13.46, 13.68, 14.08, 14.22, 14.60,
                         15.12, 15.51, 16.18, 16.65, 17.13, 17.62,
                         18.42, 18.74, 18.81, 19.25]
-
-_t = {'luminosity': _tluminosity, 'radius': _tradius, 'central temperature': _tcentral_temperature}
-evolution = pandas.DataFrame(_t, index = _tradius)
-evolution.index.name = 'time'
+                            
+evolution = pandas.DataFrame((pandas . _tluminosity , pandas . _tradius , pandas . _tcentral_temperature , pandas ._tradius ) , ( columns = [ 'luminosity' , 'radius', 'central temperature' , 'time' ] )
```
Comment (nitish705702) [inline sunpy/sun/models.py]: evolution = pandas.D
ataFrame((pandas . _tluminosity , pandas . _tradius , pandas . _tcentral_temperature , pandas ._tradius ) , ( columns = [ 'luminosity' , 'radius', 'central temperature' , 'time' ] ))

**Comment by nabobalis** [comment]
Or it could be the extra bracks around `) , (`.

**Pull request closed without merging**